lib/src/Pos/Arbitrary/Block.hs

```diff
@@ -37,6 +37,7 @@ newtype BodyDependsOnSlot b = BodyDependsOnSlot
     { genBodyDepsOnSlot :: Core.SlotId -> Gen (T.Body b)
     }
 
+
```

Comment:
Is this intentional? Part of style guide? Or just a stray change?

Reply:
Stray change.